osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/security/ReplaceKeyCommandExecutor.java

```diff
@@ -115,9 +118,8 @@ public ActionResponseDto executeBundleAction(
   private SetKeysRequestDto decryptRsaKeys(final SetKeysRequestDto setKeysRequestDto)
       throws FunctionalException {
```

Comment:
This throws clause can be removed, because in changing this.encryptionService.rsaDecrypt(setKeysRequestDto.getEncryptionKey()) into this.rsaEncrypter.decrypt(setKeysRequestDto.getAuthenticationKey()), the FunctionalException is no longer thrown (see Sonar issue here).

Reply:
I have removed the throws clause